CodeGeneration/Sources/SyntaxSupport/ExprNodes.swift

```diff
@@ -185,6 +185,51 @@ public let EXPR_NODES: [Node] = [
     ]
   ),
 
+  // the canImport expr in if config expression
+  Node(
+    name: "CanImportExpr",
+    nameForDiagnostics: "'canImport' expression",
+    kind: "Expr",
+    children: [
+      Child(
+        name: "CanImportKeyword",
+        kind: .token(choices: [.token(tokenKind: "IdentifierToken")])
```

Comment:
CI is failing in its extended syntax tree validation because this is a `canImport` keyword instead of an identifier. To reproduce the failure locally, you need to enable raw syntax validation. The easiest way to do that in Xcode is to edit `Package.swift` as follows

```diff
- if ProcessInfo.processInfo.environment["SWIFTSYNTAX_ENABLE_RAWSYNTAX_VALIDATION"] != nil {
+ if ProcessInfo.processInfo.environment["SWIFTSYNTAX_ENABLE_RAWSYNTAX_VALIDATION"] != nil || true {
```

The fix should be to change this line to 
```suggestion
        kind: .token(choices: [.keyword(text: "canImport")])
```

Reply:
Oh, I see. I didn't notice the validation is now controlled by a standalone flag rather than `DEBUG`.

Reply:
Yeah, we do that so if you build swift-syntax in debug for macros, you don’t get that extended validation, which could crash your macro. We really just want to do it in CI to catch issues in the syntax tree layout.